doc/install.text

```diff
@@ -0,0 +1,124 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
+* Install git-lfs (https://git-lfs.github.com/)
+* Install matplotlib:
+
+conda install matplotlib
+
+
+If you have a very fresh compiler version, this could lead to incompatibilities 
+with the precompiled LSST stack. In case you run into trouble compiling or testing
+the LSST stack please install your compiler in anaconda:
+
+conda install gcc
+
+
+* Create a directory for the PFS-DRP repositories:
+
+mkdir ~/stella-git
+cd ~/stella-git
+
+* Clone the relevant git repositories:
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+
+
+* Install a newer version of eigen:
+
+eups distrib install eigen 3.2.5.lsst2
+
+
+* Declare the git repositories in Linux (replace <Linux64> with <DarwinX86> for Mac OS X):
+
+eups declare obs_pfs 1.0 -c -f Linux64 -r ~/stella-git/obs_pfs
+eups declare drp_stella_data 1.0 -c -f Linux64 -r ~/stella-git/drp_stella_data
+eups declare drp_stella 1.0 -c -f Linux64 -r ~/stella-git/drp_stella
+
+
+* Setup the pipeline. As obs_pfs and drp_stella_data are required packages they are set up automatically:
+
+setup drp_stella 1.0 -v
```

Comment:
This says (lots of stuff, followed by):

```
Setting up: |eigen                          Flavor: Linux64    Version: 3.2.5.lsst2
             eigen 3.2.5.lsst1 is currently setup; overriding with 3.2.5.lsst2
Setting up: |sconsUtils                     Flavor: Linux64    Version: 12.0
Setting up: |drp_stella_data                Flavor: Linux64    Version: 1.0
             ... optional setup drp_stella_data requested by drp_stella 1.0 failed
```
Issue seems to be the hard-coded `sconsUtils` version in drp_stella_data:
```
$ setup drp_stella_data
setup: in file /ssd/swinbank/pfs/stella-git/drp_stella_data/ups/drp_stella_data.table: Product sconsUtils 12.0-1-g08b90fd not found
```
I took that out (by editing the table file) in order to continue with the instructions.